SEIR/test/data/test_ReduceR0_trivial.yml

```diff
@@ -68,7 +70,7 @@ fullrange:
   value:
     distribution: fixed
     value: 0.5
-  period_start_date: 2020-01-31
+  period_start_date: 2020-01-01
```

Comment:
note: i changed the `period_start_date` here as otherwise the `period_start_date` exceeds the `period_end_date` and the new param error check fails